src/applications/mhv-landing-page/tests/e2e/alerts/unregistered.cypress.spec.js

```diff
@@ -8,13 +8,14 @@ const viewportSizes = ['va-top-desktop-1', 'va-top-mobile-1'];
 const noHealthDataHeading = /You don’t have access to My HealtheVet/i;
 
 describe(appName, () => {
-  describe('Display content based on patient facilities', () => {
+  describe('Display content based on patient registration', () => {
```

Comment:
The underlying `va_patient` field on `vets-api` just looks at the facility count, so I'm not sure registration is closer to the truth here

Reply:
It counts associated VA treatment facilities to determine if they are registered as a VA Patient. Same difference, different avenue